modules/apps/commerce/BREAKING_CHANGES.markdown

```diff
@@ -397,4 +397,33 @@ CommercePricingClassScreenNavigationConstants;`
 
 This was moved to follow Liferay coding pattern.
 
+---------------------------------------
+
+### Destination Names Changed
+- **Date:** 2020-Sep-10
+- **JIRA Ticket:** [COMMERCE-4762](https://issues.liferay.com/browse/COMMERCE-4762)
+
+#### What changed?
+
+The prefix commerce_ has been added to the commerce destinations defined in
+`com.liferay.commerce.constants.CommerceDestinationNames`:
+
+- `liferay/commerce_order_status`;
+- `liferay/commerce_payment_status`;
+- `liferay/commerce_order_status`;
+- `liferay/commerce_payment_status`;
+- `liferay/commerce_subscription_status`.
+
+#### Who is affected?
+
+Anyone who references or uses these destinations.
+
+#### How should I update my code?
+
+Update any explicit reference to commerce destinations with the new names.
```

Comment:
Is "commerce" referencing the Commerce product/code? If so, it should be a capital C in Commerce because it is a proper noun.